pkg/scheduler/api/types.go

```diff
@@ -157,6 +157,9 @@ type ExtenderConfig struct {
 	// so the scheduler should only send minimal information about the eligible nodes
 	// assuming that the extender already cached full details of all nodes in the cluster
 	NodeCacheCapable bool
+	// Only pods with label keys and values matching this selector will be sent to the extender.
```

Comment:
You should add to the comment that this selector is applied only to filter and prioritize operations. If the extender is a binder, all the pods are sent to it for binding. Please add the comment to v1/types.go as well.